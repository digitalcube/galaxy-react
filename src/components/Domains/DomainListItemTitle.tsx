import React, { FC, PropsWithChildren } from 'react';

export type DomainListItemTitleProps = PropsWithChildren<{
  className?: string;
}>;
export const DomainListItemTitle: FC<DomainListItemTitleProps> = ({
  children,
  className,
}) => {
  return (
    <div
      className={`domain-name font-weight-bold${
        className ? ` ${className}` : ''
      }`}
    >
      {children}
    </div>
  );
};
