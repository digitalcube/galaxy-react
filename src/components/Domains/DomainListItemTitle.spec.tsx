import { render } from '@testing-library/react';
import React from 'react';
import { DomainListItemTitle } from './DomainListItemTitle';

describe('DomainListItemTitle', () => {
  it('to match snapshot', () => {
    const { asFragment } = render(
      <DomainListItemTitle>
        <a href="#">Item</a>
      </DomainListItemTitle>
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('to match snapshot [full props]', () => {
    const { asFragment } = render(
      <DomainListItemTitle className="test">
        <a href="#">Item</a>
      </DomainListItemTitle>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
