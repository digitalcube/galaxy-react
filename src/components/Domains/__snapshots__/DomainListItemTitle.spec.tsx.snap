// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DomainListItemTitle to match snapshot [full props] 1`] = `
<DocumentFragment>
  <div
    class="domain-name font-weight-bold test"
  >
    <a
      href="#"
    >
      Item
    </a>
  </div>
</DocumentFragment>
`;

exports[`DomainListItemTitle to match snapshot 1`] = `
<DocumentFragment>
  <div
    class="domain-name font-weight-bold"
  >
    <a
      href="#"
    >
      Item
    </a>
  </div>
</DocumentFragment>
`;
